src/averageAges.js

```diff
@@ -37,7 +42,18 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  let women = people.filter(person => person.sex === 'f');
+
+  if (withChildren) {
+    women
+    // eslint-disable-next-line max-len
+    = women.filter(woman => people.some(person => person.mother === woman.name));
```

Comment:
Remove the eslint-disable comment and break the line properly to avoid exceeding the max line length